core/kamon-core/src/main/scala/kamon/module/ModuleRegistry.scala

```diff
@@ -141,6 +141,8 @@ class ModuleRegistry(configuration: Configuration, clock: Clock, metricRegistry:
     val latch = new CountDownLatch(stoppedSignals.size)
     stoppedSignals.foreach(f => f.onComplete(_ => latch.countDown()))
 
+    _metricsTickerExecutor.shutdown()
+    _spansTickerExecutor.shutdown()
```

Comment:
this is making tests fail! should I separate the `stopModules` from the stopping of the `ModuleRegistry`?